main.go

```diff
@@ -256,26 +287,34 @@ func listallusers(ctx context.Context, client *github.Client, org string) ([]*gi
 }
 
 func runTrufflehog(filepath string, reponame string, orgoruser string) error {
-	outputFile1 := "/tmp/results/thog/" + orgoruser + "_" + reponame + "_" + uuid.NewV4().String() + ".txt"
+	outputDir := "/tmp/results/" + orgoruser + "/" + reponame
+	os.MkdirAll(outputDir, 0700)
+	outputFile1 := outputDir + "/" + "truffleHog"
 
 	// open the out file for writing
 	outfile, fileErr := os.OpenFile(outputFile1, os.O_CREATE|os.O_RDWR, 0644)
```

Comment:
The `outfile` basically becomes something like `/tmp/results/org/repo/truffleHog` to write the results to, correct? If somebody doesn't want to merge the output (ie the mergeOutput flag is not provided), when the looping happens over multiple users and their repos, how are those results from those files going to be written? 

The whole reason of having the `/tmp/results/thog/org_repo_uuid.txt` filepath was so that all the results from those recursively scanned repos (of multiple users and their repos) are written to their own files which would then be combined in the end. 

I am not sure I fully understand how the recursive results writing is going to happen in this case when it looks like you are writing all the results to just one file?